src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
```

Comment:
```suggestion
  const { 
    title, 
    body, 
    user,
    comments,
  } = post;
```
Such formatting might improve the readability